pkg/apis/serving/v1beta1/route_types.go

```diff
@@ -118,8 +119,14 @@ const (
 // are not generally shared.  This is defined separately and inlined so that
 // other types can readily consume these fields via duck typing.
 type RouteStatusFields struct {
-	// TODO(mattmoor): Domain (if not in addressable)
-	// TODO(mattmoor): duckv1beta1.Addressable
+	// URL holds the url that will distribute traffic over the provided traffic targets.
+	// It generally has the form {route-name}.{route-namespace}.{cluster-level-suffix}
+	// +optional
+	URL apis.URL `json:"url,omitempty"`
```

Comment:
Why is not a pointer here like in alpha?

Reply:
It should be.